python/hsfs/feature_group.py

```diff
@@ -216,6 +216,68 @@ def get_tags(self):
         """
         return self._feature_group_engine.get_tags(self)
 
+    def get_parent_feature_groups(self):
+        """Get the parent feature groups that were used to generate this feature
+        group. This is based on explicit provenance.
+        """
```

Comment:
The API documentation is based on these docstrings. Update it so that it follows the correct standard, including the documentation on the returned type.